packages/deployer/terraform/airnode/main.tf

```diff
@@ -0,0 +1,65 @@
+# TODO:
+# * switch between local and remote lambda source
+# * variable validation
+
+module "initializeProvider" {
+  source = "./modules/function"
+
+  name                = "initializeProvider"
+  handler             = "handlers/aws/index.initializeProvider"
+  source_file         = "${path.module}/../../.webpack/handlers/aws/index.js"
```

Comment:
Maybe create a variable for the webpack handlers directory (or just webpack directory?)

Reply:
Since we're planning to support providing a specific airnode version (either locally or via URL) setting a source code file felt like a better option than a whole directory. Or do you think there are some advantages in that?